### PR TITLE
[Stubhub.es] New ruleset

### PR DESCRIPTION
**Pull request by youdly**
https://github.com/EFForg/https-everywhere/pull/14269

### CODE REVIEW

**Comment by youdly** [comment]
Added `test url`.

File: src/chrome/content/rules/Stubhub.es.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="Stubhub.es">
+
+	<target host="stubhub.es" />
+	<target host="www.stubhub.es" />
+	<target host="secure.stubhub.es" />
+		<test url="http://secure.stubhub.es/robots.txt"" />
```
Comment (J0WI) [inline src/chrome/content/rules/Stubhub.es.xml]: Duped quotes

**Comment by J0WI** [comment]
Travis failed to check the hosts for some reason.

**Comment by youdly** [comment]
Rebased. let's see whether travis can access the host this time.

**Comment by J0WI** [comment]
@Hainish that's not the first time we have issues with Akamai hosts. Can we fix it somehow or should we just add it to the whitelist?

**Comment by Hainish** [comment]
This is probably due to some IP-based reputation schema blacklisting Travis IPs.  If you can test locally and it looks good, whitelist and merge.

**Comment by J0WI** [comment]
@youdly would you like to continue here?

**Comment by youdly** [comment]
whitelisted for fetch test

**Pull request merged**